src/registrar/tests/test_management_scripts.py

```diff
@@ -33,6 +33,10 @@
 logger = logging.getLogger(__name__)
 
 
+class CustomDeleteException(Exception):
+    pass
```

Comment:
**Question:** Is this pass on purpose? If so, what is the reasoning?

Reply:
This is no longer needed.  I had added a custom exception and then was testing for it, but found this was not necessary to do.

I deleted it.